comment/tests/test_api.py

```diff
@@ -273,11 +295,13 @@ def test_cannot_create_child_comment(self):
         url_data['parent_id'] = parent_id
         data = {'content': 'new child comment from api'}
         response = self.client.post(self.get_url(base_url, **url_data), data=data)
+
         self.assertEqual(response.status_code, 400)
         self.assertEqual(
             response.data['detail'],
             ContentTypeError.ID_NOT_INTEGER.format(var_name='parent', id=parent_id)
         )
+        self.assertTextTranslated(response.data['detail'], base_url)
```

Comment:
As of now, I can't seem to figure out, the reason behind the failure of this test.